plenum/test/node_request/message_request/helper.py

```diff
@@ -20,16 +21,15 @@ def split_nodes(nodes):
     return slow_node, other_nodes, primary_node, other_non_primary_nodes
 
 
-def check_pp_out_of_sync(alive_nodes, disconnected_nodes):
+def check_pp_out_of_sync(alive_nodes, disconnected_nodes, last_ordered_key):
+    def get_last_pp_key(node):
+        last_pp = node.master_replica._ordering_service.last_preprepare
+        if last_pp is None:
+            return (0, 0)
```

Comment:
should we use self.last_ordered_3pc if list_pp is None, as used in __last_pp_3pc property?